contrib/build-wine/unsign.sh

```diff
@@ -8,7 +8,7 @@ cd $here
 
 if ! which osslsigncode > /dev/null 2>&1; then
     echo "Please install osslsigncode"
-    exit
+    exit 1
 fi
 
 # exit if command fails
```

Comment:
in a similar spirit, can we put `set -e` at the top of the file?
most of our other scripts do that iirc